README.md

```diff
@@ -189,6 +189,149 @@ We only did a small refactor, so the build tab running `npm run build` should
 show no errors, and the local page under http://localhost:8080/ should look like
 it did before starting this step.
 
+## Step 3: Modeling data
+
+To build our GitHub activity feed reader, we'll eventually fetch data from this
+endpoint: [https://gh-feed.vercel.app/api](https://gh-feed.vercel.app/api).
+
+This endpoint doesn’t require API keys and works as follows:
+- It accepts two query parameters: `user` (a `string`) and `page` (an `int`).
+  For example: `https://gh-feed.vercel.app/api?user=jchavarri&page=1`
+- It returns an object (let's call it `feed`) with a key `entries`, which is an
+  array of feed entries. Each entry has the following structure:
+  - **content**: The content of the entry, usually HTML as a string.
```

Comment:
Maybe you can mention here that `content` is optional?

Reply:
fixed in https://github.com/ahrefs/react-alicante-workshop/pull/3/commits/e8d99cb2f995da3407ca96d2b36196ef9066dbfc. I also added a link to the optional types in Reason docs.